editor/editor.tsx

```diff
@@ -21,9 +21,14 @@ import './editor.css';
 import { ICONS, ICON_INFO } from './icon';
 import type { HiGlassSpec } from '@higlass.schema';
 import type { Datum } from '@gosling.schema';
+import * as ts from 'typescript';
```

Comment:
We could narrow down the scope (unsure if this will affect the size of our package though).

```suggestion
import { transpile } from 'typescript';
```